homework-solutions/js/hw-4/task-1.js

```diff
@@ -3,3 +3,13 @@
  - Создайте переменную salary со значением 1000
   - Создайте переменную grade, которая должна получить значение "middle" если salary больше или равна 1000, и значение "junior" - если меньше
   */
+
+  // Создаем переменную salary
+let salary = 1000;
+
+// Создаем переменную grade на основе условия
+let grade = salary >= 1000 ? "middle" : "junior";
```

Comment:
Используем `const`, если возможно
```suggestion
const grade = salary >= 1000 ? "middle" : "junior";
```